packages/design-system/src/components/Badge/Badge.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styles from './style.scss';
+
+function Badge(props) {
+  const badgeType = `badge--${props.type || 'primary'}`;
+
+  return (
+    <span className={`${styles.badge} ${styles[badgeType]}`}>
```

Comment:
```suggestion
className={cx(styles.badge, {
   [styles[`badge--${props.type`]]: props.type && props.type !== 'primary'
 })}
```

Reply:
Also we could add a support for custom classname from props

Reply:
I do not believe we should permit the usage of a custom class name - this would tempt devs to customise these components on their own, while we don't want them to.

Reply:
@Neob91  in fact we want to give them opportunity to customise components.